src/docs/releases/1.9.0.md

```diff
@@ -2,6 +2,18 @@
 
 Release date: Not yet released
 
+## Breaking Changes
+
+## Media Indexing
+
+Previously, `.PDF` file where auto indexed in the search providers (Elasticsearch, Lucene or Azure AI Search). Now, if you want to continue to index `.PDF` file you'll need to enable `OrchardCore.Media.Indexing.Pdf` feature.
+
+Additionally, if you needed to enable indexing for text file with `.txt`, `.md` extensions, you'll needed `OrchardCore.Media.Indexing.Text` feature.
+
+If you needed to enable indexing for other extensions like (`.docx`, or `.pptx`), you'll needed `OrchardCore.Media.Indexing.MicrosoftOffice` feature.
+
+## Change Logs
```

Comment:
Why?

Reply:
@sebastienros suggested that we split these into multiple features to allow users to specify which extensions they want to support.

Reply:
I know the intention of this PR, I mean why we have an empty Change Logs

Reply:
its not empty. under it there is a note about the new Azure Search AI